### PR TITLE
Added logging_step argument to reduce logging time

### PR DESCRIPTION
**Pull request by yeonsily**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: examples/stable-diffusion/train_text_to_image_sdxl.py [hunk 1]
```diff
@@ -1149,7 +1155,7 @@ def compute_time_ids(original_size, crops_coords_top_left):
 
                 # Gather the losses across all processes for logging (if we use distributed training).
                 avg_loss = accelerator.gather(loss.repeat(args.train_batch_size)).mean()
-                train_loss += avg_loss / args.gradient_accumulation_steps
+                train_loss = avg_loss / args.gradient_accumulation_steps
```
Comment (libinta) [inline examples/stable-diffusion/train_text_to_image_sdxl.py:1158]: if args.gradient_accumulation_steps > 1:
    train_loss += avg_loss / args.gradient_accumulation_steps
else:
    train_loss = avg_loss

File: examples/stable-diffusion/train_text_to_image_sdxl.py [hunk 2]
```diff
@@ -1175,8 +1181,6 @@ def compute_time_ids(original_size, crops_coords_top_left):
             if accelerator.sync_gradients:
                 progress_bar.update(1)
                 global_step += 1
```
Comment (libinta) [inline examples/stable-diffusion/train_text_to_image_sdxl.py:1183]: train_loss = torch.tensor(0, device='hpu')

File: examples/stable-diffusion/train_text_to_image_sdxl.py (hunk 1 above)
Comment (libinta) [inline examples/stable-diffusion/train_text_to_image_sdxl.py:1158]: can you also check what's the performance to go back to 
train_loss += avg_loss / args.gradient_accumulation_steps

File: examples/stable-diffusion/train_text_to_image_sdxl.py [hunk 3]
```diff
@@ -1204,8 +1208,12 @@ def compute_time_ids(original_size, crops_coords_top_left):
                         accelerator.save_state(save_path)
                         logger.info(f"Saved state to {save_path}")
 
-            logs = {"step_loss": loss.detach().item(), "lr": lr_scheduler.get_last_lr()[0]}
-            progress_bar.set_postfix(**logs)
+            if global_step % args.logging_step == 0:
+                train_loss_scalar = train_loss.item()
+                accelerator.log({"train_loss": train_loss_scalar}, step=global_step)
+
+                logs = {"step_loss": train_loss_scalar, "lr": lr_scheduler.get_last_lr()[0]}
```
Comment (libinta) [inline examples/stable-diffusion/train_text_to_image_sdxl.py:1215]: we can change here
if args.accumulation_steps > 1:
     logs = {"step_loss": loss.item(), "lr": lr_scheduler.get_last_lr()[0]}
else:
    logs = {"step_loss": train_loss_scalar, "lr": lr_scheduler.get_last_lr()[0]}

File: examples/stable-diffusion/train_text_to_image_sdxl.py (hunk 2 above)
Comment (yeonsily) [reply examples/stable-diffusion/train_text_to_image_sdxl.py:1184]: I put this at the end of step.

File: examples/stable-diffusion/train_text_to_image_sdxl.py [hunk 4]
```diff
@@ -1204,8 +1209,16 @@ def compute_time_ids(original_size, crops_coords_top_left):
...
+            if global_step % args.logging_step == 0:
+                train_loss_scalar = train_loss.item()
+                accelerator.log({"train_loss": train_loss_scalar}, step=global_step)
+                if args.accumulation_steps > 1:
+                    logs = {"step_loss": loss.item(), "lr": lr_scheduler.get_last_lr()[0]}
+                else:
+                    logs = {"step_loss": train_loss_scalar, "lr": lr_scheduler.get_last_lr()[0]}
+                progress_bar.set_postfix(**logs)
+
+            train_loss = zero_tensor
```
Comment (yeonsily) [inline examples/stable-diffusion/train_text_to_image_sdxl.py:1221]: I reverted 
train_loss **+=** avg_loss / args.gradient_accumulation_steps

So reset train_loss here.

**Review by libinta — APPROVED** [review]
